tests/pact/modrefs.repl

```diff
@@ -219,3 +219,5 @@
  "exercise interface with modref impl"
  true
  (modref-iface-impl.mir-impl modref-iface-impl "hi"))
+
+(typecheck 'modref-iface-impl)
```

Comment:
this won't really do anything in the context of unit tests, that's what `TypecheckSpec` and `tc.repl` are for.

Reply:
Done